### PR TITLE
Commenting the `contain_duplicates` deprecation until we have a good alternative

### PR DESCRIPTION
**Pull request by alexpop**

### CODE REVIEW

File: lib/matchers/matchers.rb [hunk 1]
```diff
@@ -88,7 +88,7 @@
 
 RSpec::Matchers.define :contain_duplicates do
   match do |arr|
-    warn '[DEPRECATION] `contain_duplicates` is deprecated and will be removed in the next major version. See https://github.com/chef/inspec/issues/738 for more details'
+    # warn '[DEPRECATION] `contain_duplicates` is deprecated and will be removed in the next major version. See https://github.com/chef/inspec/issues/738 for more details'
```
Comment (chris-rock) [inline lib/matchers/matchers.rb]: I think we should just remove the comment completely

Comment (adamleff) [reply lib/matchers/matchers.rb]: Agreed.

Comment (adamleff) [reply lib/matchers/matchers.rb]: I'm going to adopt this change now and remove the line.

**Review by adamleff — APPROVED** [review]

**Comment by alexpop** [comment]
Thank you Adam!

**Pull request merged**